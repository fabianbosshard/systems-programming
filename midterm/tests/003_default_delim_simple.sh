exec "$1" format=@b,@a,@c < tests/lorem.txt
