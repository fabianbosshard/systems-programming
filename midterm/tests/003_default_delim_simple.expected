ipsum,Lorem,dolor
tempor,eiusmod,incididunt
veniam,,minim,quis
ex,aliquip,ea
in,reprehenderit,voluptate
Excepteur,pariatur.,sint
qui,culpa,officia
